enso/experiment/finetuning.py

```diff
@@ -91,4 +79,16 @@ def predict(self, X, **kwargs):
         return predictions
 
     def cleanup(self):
-        self.model.clear()
\ No newline at end of file
+        self.model.clear()
+
+@Registry.register_experiment(ModeKeys.SEQUENCE, requirements=[("Featurizer", "PlainTextFeaturizer")])
+class FinetuneSequenceLabel(ClassificationExperiment):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.model = SequenceLabeler(autosave_path=os.path.join(RESULTS_DIRECTORY, '.autosave'), val_size=0)
```

Comment:
I'm torn as to whether we should make `val_size=0` the default in finetune or not...